client/src/views/CollectionView/CollectionView.tsx

```diff
@@ -63,16 +67,30 @@ export const CollectionView = (): ReactElement => {
         }
         return (
             <div style={{ textAlign: 'center' }}>
-                <Typography variant="h5">Edytuj kolekcję "{data.name}"</Typography>
+                {editMode && (
+                    <>
+                        <Typography variant="h5">Edytuj kolekcję "{data.name}"</Typography>
```

Comment:
może, żeby nie było łyso i tytuł kolekcji był widoczny
```suggestion
               <Typography variant="h5">Kolekcja "{data.name}"</Typography> 
               {editMode && (               
```